### PR TITLE
Card widget implementation

### PR DESCRIPTION
**Pull request by kovalandrew**

### CODE REVIEW

File: widgets/src/androidMain/kotlin/dev/icerock/moko/widgets/factory/CardViewFactory.kt [hunk 1]
```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2019 IceRock MAG Inc. Use of this source code is governed by the Apache 2.0 license.
+ */
```
Comment (Alex009) [inline widgets/src/androidMain/kotlin/dev/icerock/moko/widgets/factory/CardViewFactory.kt]: 2020 now

File: widgets/src/commonMain/kotlin/dev/icerock/moko/widgets/CardWidget.kt [hunk 2]
```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2019 IceRock MAG Inc. Use of this source code is governed by the Apache 2.0 license.
+ */
```
Comment (Alex009) [inline widgets/src/commonMain/kotlin/dev/icerock/moko/widgets/CardWidget.kt]: 2020

File: widgets/src/commonMain/kotlin/dev/icerock/moko/widgets/factory/CardViewFactory.kt [hunk 3]
```diff
@@ -0,0 +1,16 @@
+package dev.icerock.moko.widgets.factory
```
Comment (Alex009) [inline widgets/src/commonMain/kotlin/dev/icerock/moko/widgets/factory/CardViewFactory.kt]: copyright required

File: widgets/src/commonMain/kotlin/dev/icerock/moko/widgets/style/view/CornerRadiusValue.kt [hunk 4]
```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright 2019 IceRock MAG Inc. Use of this source code is governed by the Apache 2.0 license.
+ */
```
Comment (Alex009) [inline widgets/src/commonMain/kotlin/dev/icerock/moko/widgets/style/view/CornerRadiusValue.kt]: 2020

File: widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/factory/CardViewFactory.kt [hunk 5]
```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2019 IceRock MAG Inc. Use of this source code is governed by the Apache 2.0 license.
+ */
```
Comment (Alex009) [inline widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/factory/CardViewFactory.kt]: 2020

File: widgets/src/commonMain/kotlin/dev/icerock/moko/widgets/factory/CardViewFactory.kt (hunk 3 above)
Comment (kovalandrew) [reply widgets/src/commonMain/kotlin/dev/icerock/moko/widgets/factory/CardViewFactory.kt]: added

File: widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/factory/CardViewFactory.kt (hunk 5 above)
Comment (kovalandrew) [reply widgets/src/iosMain/kotlin/dev/icerock/moko/widgets/factory/CardViewFactory.kt]: fixed

File: widgets/src/commonMain/kotlin/dev/icerock/moko/widgets/style/view/CornerRadiusValue.kt (hunk 4 above)
Comment (kovalandrew) [reply widgets/src/commonMain/kotlin/dev/icerock/moko/widgets/style/view/CornerRadiusValue.kt]: fixed

File: widgets/src/commonMain/kotlin/dev/icerock/moko/widgets/CardWidget.kt (hunk 2 above)
Comment (kovalandrew) [reply widgets/src/commonMain/kotlin/dev/icerock/moko/widgets/CardWidget.kt]: fixed

File: widgets/src/androidMain/kotlin/dev/icerock/moko/widgets/factory/CardViewFactory.kt (hunk 1 above)
Comment (kovalandrew) [reply widgets/src/androidMain/kotlin/dev/icerock/moko/widgets/factory/CardViewFactory.kt]: fixed

**Comment by Alex009** [comment]
implementation of #37

**Pull request merged**